SunEngine.Commons/Models/CacheSettings.cs

```diff
@@ -0,0 +1,18 @@
+namespace SunEngine.Commons.Models
+{
+    public enum CachePolicy
+    {
+        AlwaysPolicy = 0,
```

Comment:
Do we need 0 here? Isn't it default?

Reply:
Yes, It's just a habit)

Reply:
I would suggest to remove it then, I confuse the reader a bit.